### PR TITLE
FEI-4957.8: Remove Flow related plugins, settings, and lib defs

### PR DESCRIPTION
**Pull request by kevinbarabash**
## Summary:
This removes all Flow relate things.

NOTE: `eslint` is currently not running on .ts files.  I'll fix that in the next PR.

Issue: FEI-4957

## Test plan:
Let CI run the checks

### CODE REVIEW

File: packages/eslint-config-khan/index.js [hunk 1]
```diff
@@ -72,30 +69,6 @@ module.exports = {
         "template-curly-spacing": OFF, // TODO(kevinb): enable this
         "valid-jsdoc": OFF, // TODO(kevinb): Enable this since we are using jdocs in some places.
 
-        /**
```
Comment (jeresig) [inline packages/eslint-config-khan/index.js:75]: Are there TS eslint rules we want to be enabling?

**Review by jeresig — APPROVED** [review]
Thanks for cleaning all of this up!

File: packages/eslint-config-khan/index.js (hunk 1 above)
Comment (kevinbarabash) [reply packages/eslint-config-khan/index.js:75]: Yes, the migration guide has a mapping from Flow rules to TypeScript rules.  I'll add those to the config in the next PR when I enable eslint to check .ts files.

Comment (jeresig) [reply packages/eslint-config-khan/index.js:75]: Great, ok!

**Pull request merged**